addons/rearm/functions/fnc_addVehicleMagazinesToSupply.sqf

```diff
@@ -29,16 +29,17 @@ if (_vehicle isEqualType objNull) then {_vehicle = typeOf _vehicle};
 if (_vehicle == "") exitWith {
     ERROR_1("VehicleType [%1] is empty in ace_rearm_fnc_addVehicleMagazinesToSupply",_string);
 };
+private _turrets = [_vehicle] call FUNC(getAllRearmTurrets);
```

Comment:
Nobody forces you to like macros. They work as documented. They are used everywhere in ACE, CBA and many other projects. Just because somewhere it is useful to use macros not functions.